ot-node.js

```diff
@@ -137,19 +137,30 @@ class OTNode {
         }
     }
 
-    async createProfile() {
+    async createProfiles() {
         const blockchainModuleManager = this.container.resolve('blockchainModuleManager');
-        if (!blockchainModuleManager.identityExists()) {
-            const networkModuleManager = this.container.resolve('networkModuleManager');
-            const peerId = networkModuleManager.getPeerId();
-            await blockchainModuleManager.deployIdentity();
-            await blockchainModuleManager.createProfile(peerId);
-            if (process.env.NODE_ENV !== 'development' && process.env.NODE_ENV !== 'test') {
-                this.saveIdentityInUserConfigurationFile(blockchainModuleManager.getIdentity());
+        const createProfilesPromises = Object.keys(
+            this.config.modules.blockchain.implementation,
+        ).map(async (blockchain) => {
+            if (!blockchainModuleManager.identityExists(blockchain)) {
+                const networkModuleManager = this.container.resolve('networkModuleManager');
+                const peerId = networkModuleManager.getPeerId();
+                await blockchainModuleManager.deployIdentity(blockchain);
+                await blockchainModuleManager.createProfile(blockchain, peerId);
```

Comment:
Add error handling
If error is thrown we should remove that implementation from blockchainModule
If 0 profiles are created then stop the node